types/node/test/crypto.ts

```diff
@@ -356,6 +356,12 @@ import { promisify } from 'util';
     crypto.randomFill(i32arr, 2, 3, (err: Error | null, buf: Int32Array) => void {});
 }
 
+{
```

Comment:
This should be in the v14 folder as well.

Reply:
Missed that. Fixed.

Thanks.